java/defs.bzl

```diff
@@ -25,6 +25,8 @@ load("//java/toolchains:java_package_configuration.bzl", _java_package_configura
 load("//java/toolchains:java_runtime.bzl", _java_runtime = "java_runtime")
 load("//java/toolchains:java_toolchain.bzl", _java_toolchain = "java_toolchain")
 
+version = "7.12.0"
```

Comment:
no longer needed, please revert

Reply:
Done! Is this needed in the future? Or is it just this time only? @hvadehra

Reply:
Not needed ever :)